src/common/hooks/usePaginatedPieces/hooks/usePieceItemsFetch.js

```diff
@@ -8,10 +8,10 @@ import {
 export const usePieceItemsFetch = ({ instanceId, tenantId }) => {
   const ky = useOkapiKy({ tenant: tenantId });
 
-  const fetchPieceItems = ({ pieces, crossTenant, signal }) => {
+  const fetchPieceItems = ({ pieces, isConsortium, signal }) => {
     const kyExtended = ky.extend({ signal });
 
-    return crossTenant
+    return isConsortium
```

Comment:
The word `crossTenant` was used specifically to indicate the need to obtain data from several tenants of the consortium. The fact that the user is in ECS mode does not require data loading from several tenants.